src/components/DepositWidget/index.tsx

```diff
@@ -335,6 +335,39 @@ interface BalanceDisplayProps extends TokenLocalState {
   ): Promise<void>
 }
 
+const customFilterFnFactory = (customStopCheck: (...any: unknown[]) => boolean) => (searchTxt: string) => ({
+  symbol,
+  name,
+  address,
+}: TokenBalanceDetails): boolean => {
+  if (
+    customStopCheck &&
```

Comment:
Why do you need this check? From what I understand this param is not optional, so it should always be true, no?

Reply:
good catch, this is pointless what im doing, fixed it